src/main/kotlin/com/openlattice/shuttle/MissionControl.kt

```diff
@@ -350,11 +350,23 @@ class MissionControl(
                             mutableSetOf(),
                             organizationId
                     )
-                    val entitySetId = entitySetsApi
-                            .createEntitySets(setOf(entitySet))
-                            .getValue(entityDefinition.entitySetName)
-                    check(entitySetId == entitySet.id) { "Submitted entity set id does not match return." }
-                    entitySets[entityDefinition.entitySetName] = entitySet
+                    entitySets[entityDefinition.entitySetName] = try {
```

Comment:
This should only affect locally tested integrations, yeah?

Reply:
Correct, unless someone changed elsewhere.

Reply:
yeah